### DISCUSSION TITLE
gcc compilation error for Kube tool on  MacOS Big Sur

### DISCUSSION THREAD

**Discussion by shaheenzia** [Homebrew/discussions#855] [Getting started]
After upgrading to MacOS to Big Sur, getting following error for Kube tool installation

brew install aci/tools/kcli
==> Installing kcli from aci/tools
==> Downloading https://artifacts.apple.com/aci-binaries-local/com/apple/aci/kcli/1.1.1/kcli-1.1.1.tar.gz
Already downloaded: /Users/shaheen/Library/Caches/Homebrew/downloads/aaa5c94e4526f6b8fb69a92d4075059f4c827c8c28c0a09e9ff327d190639b2a--kcli-1.1.1.tar.gz
xcode-select: error: malformed developer path ("/Library/Developer/CommandLineTools")
Error: An exception occurred within a child process:
  CompilerSelectionError: aci/tools/kcli cannot be built with any available compilers.
Install GNU's GCC:
  brew install gcc

$ brew install gcc
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 2 taps (homebrew/core and homebrew/cask).
==> Updated Formulae
Updated 2 formulae.
==> Updated Casks
Updated 1 cask.

Error: gcc: the bottle needs the Xcode CLT to be installed.You can try to install from source with:
  brew install --build-from-source gcc
Please note building from source is unsupported. You will encounter build
failures with some formulae. If you experience any issues please create pull
requests instead of asking for help on Homebrew's GitHub, Twitter or any other
official channels.

**Comment by carlocab** [comment]
Install the CLT with
```
xcode-select --install
```

**Reply by shaheenzia** [reply]
Thanks, install Xcode worked .. it could be nice if error say run this command to fix it.

**Reply by GavinSyria** [reply]
thanks so much !! this works <3
